modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3773,7 +3516,7 @@ Request::
 [source.wrap,bash]
 ----
 curl -X POST --user tigergraph:tigergraph \
-  --url 'http://localhost:14240/gsql/library?functionName=gds.community.louvain&graph=poc_graph' \
+  --url 'http://localhost:14240/gsql/v1/gds.community.louvain?graph=poc_graph' \
```

Comment:
should be `"/gsql/v1/library/gds.community.louvain"`?

Reply:
Done